azure/datalake/store/core.py

```diff
@@ -168,10 +168,10 @@ def du(self, path, total=False, deep=False):
         else:
             return {p['name']: p['length'] for p in files}
 
-    def df(self):
-        """ Resource summary
-        """
-        return self.azure.call('GETCONTENTSUMMARY')['ContentSummary']
+    def df(self, path):
+        """ Resource summary of path """
+        path = AzureDLPath(path).trim()
+        return self.azure.call('GETCONTENTSUMMARY', path.as_posix())['ContentSummary']
```

Comment:
Do we have any other calls that don't take in path? I just want to make sure that we take in path everywhere that it is supported (which should be for all the calls, I think).